data/mods/ssb/moves.ts

```diff
@@ -382,6 +382,68 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "???",
 	},
 
+// Jett x_x
+	thehuntison: {
+		accuracy: 100,
+		basePower: 55,
+		category: "Physical",
+		desc: "If an opposing Pokemon switches out this turn, this move hits that Pokemon before it leaves the field, even if it was not the original target. If the user moves after an opponent using Parting Shot, U-turn, or Volt Switch, but not Baton Pass, it will hit that opponent before it leaves the field. Power doubles and no accuracy check is done if the user hits an opponent switching out, and the user's turn is over; if an opponent faints from this, the replacement Pokemon does not become active until the end of the turn. Raises the user's Attack by 2 stages if this move KOes the target.",
+		shortDesc: "Foe: 2x power when switching. +2 Atk if KO.",
+		name: "The Hunt is On!",
+		pp: 15,
+		priority: 0,
+		flags: {contact: 1, protect: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Sucker Punch', target);
+			this.add('-anim', source, 'Pursuit', target);
+		},
+		beforeTurnCallback(pokemon) {
+			for (const side of this.sides) {
+				if (side === pokemon.side) continue;
+				side.addSideCondition('thehuntison', pokemon);
```

Comment:
why cant you just use the pursuit volatile?

Reply:
does it show in display anywhere if i just use the pursuit volatile?

Reply:
OK I WAS WRONG

Reply:
lol